aea/skills/base.py

```diff
@@ -308,7 +308,11 @@ def parse_module(
         behaviour_module = load_module("behaviours", Path(path))
         classes = inspect.getmembers(behaviour_module, inspect.isclass)
         behaviours_classes = list(
-            filter(lambda x: re.match("\\w+Behaviour", x[0]), classes)
+            filter(
+                lambda x: re.match("\\w+Behaviour", x[0])
+                and not str.startswith(x[1].__module__, "aea."),
```

Comment:
without this clause, also behaviour classes imported from `aea` like `TickerBehaviour` would have matched.